go.mod

```
@@ -47,25 +52,24 @@ require (
 	golang.org/x/mod v0.30.0
 	golang.org/x/text v0.31.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.32.3
-	k8s.io/apimachinery v0.32.3
-	k8s.io/client-go v0.32.3
-	k8s.io/cluster-bootstrap v0.32.3
-	k8s.io/component-base v0.32.3
+	k8s.io/api v0.33.3
+	k8s.io/apimachinery v0.33.3
+	k8s.io/client-go v0.33.3
+	k8s.io/cluster-bootstrap v0.33.3
+	k8s.io/component-base v0.33.3
 	k8s.io/klog/v2 v2.130.1
 	k8s.io/kubectl v0.32.3
 	k8s.io/utils v0.0.0-20250321185631-1f6e0b77f77e
 	sigs.k8s.io/cloud-provider-azure v1.32.3
```

Let's also bump this to stay in sync with the other k8s.io dependencies.
```suggestion
	sigs.k8s.io/cloud-provider-azure v1.33.6
```